src/views/admin/RolleCreationView.vue

```diff
@@ -153,8 +168,12 @@
 
   const translatedCreatedRolleMerkmale: ComputedRef<string> = computed(() => {
     // Check if `createdRolle.merkmale` exists and is an array
-    if (!rolleStore.createdRolle?.merkmale || !Array.isArray(rolleStore.createdRolle.merkmale)) {
-      return '';
+    if (
+      !rolleStore.createdRolle?.merkmale ||
+      !Array.isArray(rolleStore.createdRolle.merkmale) ||
+      rolleStore.createdRolle.merkmale.size == 0
```

Comment:
Woher kommt das .size? Was spricht gegen .length? Zur Typensicherheit bitte auch === benutzen

```suggestion
      rolleStore.createdRolle.merkmale.length === 0
```

Reply:
Merkmale und systemrechte sind ein Set und daher muss der size evaluiert werden statt length (bei arrays)